test/mocha/get-timeframes.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-env mocha */
+
+import should from 'should';
+
+import config from '../test-config.js';
+import { openDb, closeDb, importGtfs, getTimeframes } from '../../index.js';
+
+describe('getTimeframes():', () => {
+  before(async () => {
+    openDb(config);
+    await importGtfs(config);
+  });
+
+  after(() => {
+    const db = openDb(config);
+    closeDb(db);
+  });
+
+  it('should return empty array if no timeframes', () => {
+    const timeframeGroupId = 'not_real';
+
+    const results = getTimeframes({
+      timeframe_group_id: timeframeGroupId,
+    });
+    should.exists(results);
+    results.should.have.length(0);
+  });
```

Comment:
It looks like the included test dataset does not include `timeframes.txt`, so I was not able to write a test that confirms the actual database output of `getTimeframes`.